src/app/index.html

```diff
@@ -87,13 +87,12 @@
       <div class="content--row_1 register">
         <div></div>
         <div class="register--content">
-          <h1>INTRODUCING OPENSHIFT.IO</h1>
+          <h1>WELCOME TO OPENSHIFT.IO</h1>
           <p class="introduction-text">
             OpenShift.io is an open online development environment for planning, creating and deploying hybrid cloud services.
           </p>
-          <a class="btn btn-heavy-cta btn-lg beta-submit uppercase" role="button" id="registerContent">sign up for the preview</a>
+          <a class="btn btn-heavy-cta btn-lg beta-submit uppercase" role="button" id="registerContent">sign up today</a>
           <div class="help-block">
```

Comment:
can you remove this `<div>` if the content is being deleted?

Reply:
Good question. @joshuawilson suggestions on this?

Reply:
@MikGue I'm just looking for you to update your PR to remove the empty `<div>`. Removing it won't cause any issues.

Reply:
Gotcha. It's been a while since I was in Github, so please bare with me. Are you referring to the one on line 95?

Reply:
No problem - yes - the `<div class="help-block">` on line 95 and it's closing `</div>`.